pom.xml

```
@@ -6,12 +6,12 @@
     <parent>
         <groupId>org.carlspring.strongbox</groupId>
         <artifactId>strongbox-parent</artifactId>
-        <version>1.0-SNAPSHOT</version>
+        <version>1.0-PR-60-SNAPSHOT</version>
     </parent>
     
     <artifactId>strongbox-db</artifactId>
     <packaging>pom</packaging>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-PR-16-SNAPSHOT</version>
```

Version is incorrect - should be `18`
```suggestion
    <version>1.0-PR-18-SNAPSHOT</version>
```

---

fixed